### ISSUE TITLE
Language.save_to_directory fails if the path is a string

### ISSUE THREAD

**Issue by jgc128** [explosion/spaCy#1009]
Hi,

The documentation says that the `path` argument in the `Language.save_to_directory` method can be a string. However, this methods fails in such case:
```
File "lib/python3.6/site-packages/spacy/language.py", line 353, in save_to_directory
    self.setup_directory(path, **configs)
File "lib/python3.6/site-packages/spacy/language.py", line 192, in setup_directory
    directory = path / name
TypeError: unsupported operand type(s) for /: 'str' and 'str'
```

It works fine if the argument is a pathlib's Path. 

## Your Environment

* **spaCy version:** 1.8.0
* **Platform:** Linux-4.4.0-66-generic-x86_64-with-Ubuntu-16.04-xenial
* **Python version:** 3.6.1
* **Installed models:** en, en_core_web_md

**Comment by ines** [comment]
Hmm, this is interesting! Could you share how you're calling `save_to_directory()`, i.e. the value you're passing in?

spaCy currently uses an [`ensure_path()` helper function](https://github.com/explosion/spaCy/blob/master/spacy/util.py#L43) that should make sure string types are turned into pathlib paths. But somehow, this seems to fail here and returns a string...

**Comment by jgc128** [comment]
Oh, I see - I was using the version from pip, and it does not call `ensure_path` in `save_to_directory`, whereas the version from github uses it. 

I'll close the issue then.

Thanks!

**Comment by ines** [comment]
Oh, I didn't realise this was a fix we hadn't released as part of v1.8.0 yet. The next point release will hopefully be up on pip later tonight! Sorry about the confusion.
